app/grandchallenge/core/templatetags/bleach.py

```diff
@@ -38,12 +67,24 @@ def md2email_html(markdown: str | None):
     )
 
 
+@register.filter
+def md2page_html(markdown: str | None):
+    """Converts markdown to clean html intended for showing as a page"""
```

Comment:
I don't really understand why we need a new filter here, in what situations would I use each of:

- `md2html`: this was for all rendering on the site, apart from emails
- `md2html_email`: this was for rendering for emails
- `md2html_page`: don't really get it, `Page` objects do not use markdown and this seems to be used everywhere

Reply:
I imagine them roughly for:

- `md2html`: all rendering without a specific filter, sane defaults. Generally used for things that leave GC (i.e. go to CIRRUS).
- `md2html_email`: rendering emails, email specific HTML features (should really remove the BS4 additions here)
- `md2page_html`: rendering information pages to be shown as information or blog or documentation

Reply:
I still don't get why we need the new split?

Reply:
No longer relevant after the refactor.